api/src/config/passport.js

```diff
@@ -35,36 +36,71 @@ module.exports = function setupPassport(csrf) {
             }
         )
     );
-    // passport.use(
-    //     new FacebookStrategy(
-    //         {
-    //             clientID: process.env.FACEBOOK_APP_ID,
-    //             clientSecret: process.env.FACEBOOK_APP_SECRET,
-    //             callbackURL: 'http://localhost:3000/v1/auth/facebook/callback',
-    //         },
-    //         (accessToken, refreshToken, profile, cb) => {
-    //             cb(null, profile);
-    //         }
-    //     )
-    // );
+    passport.use(
+        new FacebookStrategy(
+            {
+                clientID: process.env.FACEBOOK_APP_ID,
+                clientSecret: process.env.FACEBOOK_APP_SECRET,
+                callbackURL: `${process.env.BACK_URI}/v1/auth/facebook/callback`,
+                profileFields: ['id', 'first_name', 'last_name', 'picture'],
+            },
+            (accessToken, refreshToken, profile, cb) => {
+                // console.log('profile =', profile);
+                const { provider, name, id } = profile;
```

Comment:
We should write try/catch blocks on each callback of that kind to prevent unhandled errors.